content/en/docs/setup/install/multicluster/shared/index.md

```diff
@@ -216,6 +212,12 @@ Apply the primary cluster's configuration.
 $ istioctl install -f istio-main-cluster.yaml --context=${MAIN_CLUSTER_CTX}
 {{< /text >}}
 
+If you're using the `istio-ingressgateway` option, expose the control plane by using the provided sample configuration.
```

Comment:
What to do if you are NOT using the `istio-ingressgateway` option. I don't think that's possible. How about?
```suggestion
Expose the control plane by configuring the `istio-ingressgateway` using the provided sample configuration.
```

Reply:
In the other case (Internal Load Balancer), the control plane is already exposed by the `components` part of the install config

Reply:
I see. Maybe this would be a little more clear?
```suggestion
If you selected the `istio-ingressgateway` option, expose the control plane using the provided sample configuration.
```